tests/test_realm.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from anys import  ANY_STR
+from test_helpers import (send_JSON_command, subscribe, read_JSON_message)
+
+@pytest.mark.asyncio
+async def test_realm_realmCreated(websocket, context_id, html):
+    url = html()
+
+    await subscribe(websocket, "script.realmCreated")
+
+    await send_JSON_command(
+        websocket, {
+            "method": "browsingContext.navigate",
+            "params": {
+                "context": context_id,
+                "url": url,
+                "wait": "complete",
+            }
+        })
+    
+    response = await read_JSON_message(websocket),
+
+    # Wait for `script.realmCreated` event.
+    assert {
+        "method": "script.realmCreated",
+        "params": {
+            "type": "window",
+            "origin": "null",
+            "realm": ANY_STR,
+            "context": context_id,
+        }
+    } == response[0]
```

Comment:
Help needed here. The response is a tuple with 1 element.

Reply:
See suggestion below